setup.py

```diff
@@ -33,7 +33,7 @@ def get_all_files_in_dir(directory, pattern):
 rgb_files = get_all_files_in_dir('acme_diags/plot/colormaps', '*.rgb')
 control_runs_files = get_all_files_in_dir('acme_diags/driver/control_runs', '*.csv')
 
-INSTALL_PATH = os.path.join(sys.prefix, 'share/e3sm_diags/')
```

Comment:
@zshaheen, you can't use the current `sys.prefix` as the install path because the package is built in one environment (and usually on a completely different machine) than the one where it will be installed.  So if I build the package in my base environment, the scripts also get installed in `shared/e3sm_diags` in my base environment, which is not what I want and the package is broken for everyone else because they don't have a `/home/xylar/miniconda3/` path.  The solution is just to leave it as a relative path, and it will automatically have the install prefix prepended.